ocr_prediction/ocr_prediction.py

```diff
@@ -0,0 +1,32 @@
+from pytesseract import image_to_string
+from PIL import Image
+
```

Comment:
This is a neat idea! I think this could actually be a useful feature to add - let's stick this in the "feature_generation" package (you can leave it as a separate file, imported with __init__.py)